source/frontend/src/features/properties/map/lease/common/LeaseHeaderTenants.tsx

```diff
@@ -0,0 +1,31 @@
+import ExpandableTextList from 'components/common/ExpandableTextList';
+import { ILease } from 'interfaces';
+import { Api_Person } from 'models/api/Person';
+import * as React from 'react';
+import { formatNames } from 'utils/personUtils';
+
+export interface ILeaseHeaderTenantsProps {
+  lease?: ILease;
+  delimiter?: React.ReactElement | string;
+  maxCollapsedLength?: number;
+}
+
+export const LeaseHeaderTenants: React.FC<ILeaseHeaderTenantsProps> = ({
+  lease,
+  delimiter = '; ',
+  maxCollapsedLength = 2,
+}) => {
+  return (
+    <ExpandableTextList<Api_Person>
+      items={lease?.persons ?? []}
+      keyFunction={(p: Api_Person, index: number) => `lease-tenant-${p?.id ?? index}`}
+      renderFunction={(p: Api_Person) => (
+        <>{formatNames([p?.firstName, p?.middleNames, p?.surname])}</>
```

Comment:
There is no confluence page for this story that I could see, but noticed that in the mockup all of the names were formatted as last, first ie: Smart, Jenny

This seems out of line with what we've discussed in the past and decided upon for names on other screens, such as contacts. Worth checking with Bobbi and then putting a note on the story so we don't get a bug for this.

Reply:
I took this code verbatim from old Lease Header. I would expect this story would be mostly about UI re-arrangement, not "logic changes". I'll ask Bobbi anyways but would lean to keep the existing logic (in the new UI) and file a change request if needed.